### PR TITLE
Add documentation for filtering on multiple health scores

### PR DESCRIPTION
**Pull request by martijnbleeker**

### CODE REVIEW

File: api-docs/swagger-v1.1.yml [hunk 1]
```diff
@@ -1096,6 +1106,12 @@ parameters:
     description: restrict by health score
     enum: ['A', 'B1', 'B2', 'C1', 'C2']
     required: false
+  personal_health_scores:
+    name: personal_health_scores[]
+    type: string
+    in: query
+    description: restrict by health score, repeat parameter
+    required: false
```
Comment (wvengen) [inline api-docs/swagger-v1.1.yml]: I would put in the enum here. We may need to use a different way to accept array parameters (e.g. like [here](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#path-item-object-example)), need to experiment with that if the client supports it.

**Review by wvengen — CHANGES_REQUESTED** [review]

File: api-docs/swagger-v1.1.yml [hunk 2]
```diff
@@ -1108,10 +1108,14 @@ parameters:
...
     name: personal_health_scores[]
-    type: string
+    type: array
+    items:
+      type: string
+      enum: ['A', 'B1', 'B2', 'C1', 'C2']
     in: query
     description: restrict by health score, repeat parameter
     required: false
+    collectionFormat: multi
```
Comment (wvengen) [inline api-docs/swagger-v1.1.yml]: Cool, curious to see how it looks in the UI :) :+1:

Comment (martijnbleeker) [reply api-docs/swagger-v1.1.yml]: I've tested it locally, it works. Not sure about the look though.

screen shot 2017-03-06 at 14 22 13

Comment (wvengen) [reply api-docs/swagger-v1.1.yml]: Interesting, but that's it. I think we can omit the _, repeat parameter_ when using this :)

File: api-docs/swagger-v1.1.yml [hunk 3]
```diff
@@ -1113,7 +1113,7 @@ parameters:
       type: string
       enum: ['A', 'B1', 'B2', 'C1', 'C2']
     in: query
-    description: restrict by health score
+    description: restrict by health scores
```
Comment (wvengen) [inline api-docs/swagger-v1.1.yml]: (and, naturally, the others too)

**Review by wvengen — APPROVED** [review]

**Pull request merged**